data/json/items/ammo.json

```diff
@@ -794,7 +812,7 @@
     "description": "Copper bits.  Could be used to craft something, for example makeshift shotgun shells.",
     "material": [ "copper" ],
     "volume": "250 ml",
```

Comment:
Does this volume refer to the stack of 200 or an individual item?

Reply:
Volume refers to the stack of 200.